app/main.py

```diff
@@ -2,0 +1,18 @@
+from app.cafe import Cafe
+from app.errors import NotWearingMaskError, VaccineError
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
+    masks_needed = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            masks_needed += 1
+
+    if masks_needed:
+        return f"Friends should buy {masks_needed} masks"
```

Comment:
According to the checklist, we don't need to compare a variable to `0` if there is no need. Here, you are checking if `masks_needed` is not zero. You can simplify this by just checking `if masks_needed:`. Python will interpret any non-zero value as `True`.